src/main/java/com/iota/iri/storage/Persistable.java

```diff
@@ -2,10 +2,49 @@
 
 import java.io.Serializable;
 
+/**
+ * Any object that can be serialized and stored in the DB.
```

Comment:
```suggestion
 * Any object that can be serialized and stored in the database.
```

You write database everywhere else, should keep this constant